Sources/AppcuesKit/AppcuesKit.docc/AnchoredTooltips.md

```diff
@@ -0,0 +1,51 @@
+# Configuring Views for Anchored Tooltips
+
+The Appcues iOS SDK supports anchored tooltips targeting views built with `UIKit` or `SwiftUI`.
+
+Instrumenting your application views as described below allows the Appcues iOS SDK to create a mobile view selector. This selector is used by the Appcues Mobile Builder to create and target anchored tooltips. When a user qualifies for a flow, this selector is used to render the anchored tooltip content in the correct location.
+
+## Instrumenting UIKit Views
+
+The following `UIView` properties are used to identify elements, in order of precedence:
```

Comment:
I could not see a way to link Apple class references like `UIView` to the DocC page directly with the double backtick syntax. In items below, I used web URL, which does seem to actually link directly into the doc page inside of the doc browser when available, but did not do this for every reference.

Reply:
Only symbols in the AppcuesKit module can be linked with double backticks.